docs/docs/usage/scaladoc/settings.md

```diff
@@ -117,3 +117,19 @@ A base URL to use as prefix and add `canonical` URLs to all pages. The canonical
 ##### -siteroot
 
 A directory containing static files from which to generate documentation. Default directory is `./docs`
+
+##### -versions-dictionary-url
+
+A URL pointing to a JSON document containing a dictionary version -> documentation location.
+The JSON file has single property "versions" that holds dictionary of labels of specific docs and URL pointing to their index.html top-level file.
```

Comment:
```suggestion
The JSON file has single property `versions` that holds the dictionary associating the labels of specific versions of the documentation to the URLs pointing to their index.html
```

Its argument must be a URL to a JSON document holding information about the locations of specific versions.